e2e/README.md

```diff
@@ -135,6 +135,7 @@ We are going to enumerate the variables that will affect the product versions us
 The following environment variables affect how the tests are run in both the CI and a local machine.
 
 - `SKIP_SCENARIOS`: Set this environment variable to `false` if it's needed to include the scenarios annotated as `@skip` in the current test execution. Default value: `true`.
+- `BEATS_LOCAL_PATH`: Set this environment variable to the path to your local copy of Beats if it's needed to use the binary snapshots produced by your local build instead of the official releases. The snapshots will be fetched from the `build/distributions` local directory. Default: empty. This variable is intented to be used by Beats developers, when testing locally the artifacts generated its own build.
```

Comment:
Nit: The default should be moved to the very end to make it consistent with the rest.

Reply:
is this path to beats or beats artifacts/packages

Reply:
It's to the base dir of the Beats project. I'll clarify it in the section

Reply:
> Nit: The default should be moved to the very end to make it consistent with the rest.

I'm going to change L139 too, as aI used that as example.